lib/buffer.js

```diff
@@ -264,8 +264,8 @@ function fromObject(obj) {
   }
 
   if (obj) {
-    if (isArrayBuffer(obj.buffer) || 'length' in obj ||
-        isSharedArrayBuffer(obj)) {
+    if ('length' in obj || isArrayBuffer(obj.buffer) ||
```

Comment:
@ojss If it doesn't happen here, don't sweat it. Removing `in` isn't directly tied to this PR.